### DISCUSSION TITLE
ARM-DOE Github Tutorial

### DISCUSSION THREAD

**Discussion by mgrover1** [ProjectPythia/projectpythia.github.io#216] [General]
Hey ya'll!

I am helping with a Github Tutorial next week for some of the ARM user community, and figured we could repurpose some of the amazing Github content that has been merged recently...

The plan would be to create a new repo under `ProjectPythiaTutorials`... it would be our first non-NCAR organized tutorial hosted there, and an opportunity to collaborate with the DOE sphere.

The main goals of the tutorial are:
* Familiarize users with Github
* Fork the [repo used for collaboration](https://github.com/ARM-Development/Aerosol-Mentors-Processing-Routines)
* Clone the repo to their local machine (supporting both Windows/Mac users)
* Make a change + commit
* Submit a pull request

Open to thoughts/suggestions! 

Here is the repo we are using - https://github.com/ProjectPythiaTutorials/github-arm-2022-02-28

**Comment by dcamron** [comment]
Welcome back, Max! Long time no see 😜 

This is really exciting and I'm super glad to see a tutorial on GitHub being assembled. Will the synchronous session of this tutorial be publicly available and/or the recording be shareable to the Pythia YouTube channel afterwards?

**Reply by mgrover1** [reply]
The recording will be available after!

**Comment by kmpaul** [comment]
This is awesome, @mgrover1!  I think your plan is a good one.  I can't think of a reason not to just charge forward.
